vms/platformvm/network/network.go

```diff
@@ -131,29 +240,13 @@ func (n *network) IssueTx(ctx context.Context, tx *txs.Tx) error {
 	}
 
 	txID := tx.ID()
-	n.gossipTx(ctx, txID, msgBytes)
-	return nil
+	n.legacyGossipTx(ctx, txID, msgBytes)
+	n.txPushGossiper.Add(tx)
+	return n.txPushGossiper.Gossip(ctx)
 }
 
 // returns nil if the tx is in the mempool
 func (n *network) issueTx(tx *txs.Tx) error {
-	txID := tx.ID()
-	if _, ok := n.mempool.Get(txID); ok {
-		// The tx is already in the mempool
-		return nil
-	}
-
-	// Verify the tx at the currently preferred state
-	if err := n.txVerifier.VerifyTx(tx); err != nil {
-		n.log.Debug("tx failed verification",
-			zap.Stringer("txID", txID),
-			zap.Error(err),
-		)
-
-		n.mempool.MarkDropped(txID, err)
-		return err
-	}
-
 	// If we are partially syncing the Primary Network, we should not be
 	// maintaining the transaction mempool locally.
 	if n.partialSyncPrimaryNetwork {
```

Comment:
I think we should probably not be sending any pull gossip requests if the node is partial synced as well.